app/src/main/java/org/openobservatory/ooniprobe/utils/Alert.java

```diff
@@ -18,4 +18,18 @@ public void onClick(DialogInterface dialog, int id) {
                     }
                 }).show();
     }
+
+    public static void alertDialogTwoButtons(Context c, String title, String text, DialogInterface.OnClickListener ocl) {
```

Comment:
It is unused, I prefer to integrated directly into the notificationdialog. But it will be useful to run dialogs with two button, Alert is just an helper class